service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/bqdataset/ControlledBigQueryDatasetResource.java

```diff
@@ -301,11 +293,8 @@ public boolean partialEqual(Object o) {
 
   @Override
   public int hashCode() {
-    return new HashCodeBuilder(17, 37)
-        .appendSuper(super.hashCode())
-        .append(datasetName)
-        .append(projectId)
-        .toHashCode();
+    return Objects.hashCode(
+        super.hashCode(), datasetName, projectId, defaultTableLifetime, defaultPartitionLifetime);
   }
```

Comment:
Should the `equals` and `hashCode` methods for `ControlledFlexibleResource` be updated as well?

Reply:
We should standardize on one library for equals and hash, but it doesn't affect correct operation.
One thing I note is that with the conversion of `equals` to `partialEquals` we do not have `equals` methods on many of the classes. So maybe we can pick a consistent equals/hash implementation and make a ticket to apply it everywhere.